phantom-load.js

```diff
@@ -55,4 +59,16 @@ function renderPage(url) {
   })
 }
 
+function setCookies(page, jar) {
+  var cookieJar = JSON.parse(jar)
```

Comment:
Wanted to note - `JSON.parse` will throw an error on failure.
This would cause an `exit(1)` failure which is good. Will prevent us from sending a bad cookie string to the process.